src/main/java/ReviewResponse.java

```diff
@@ -13,6 +16,8 @@ public class ReviewResponse {
     @Getter
     String okbb;
 
+    @Setter
```

Comment:
Using Lombok's @Setter annotation can lead to violating the Encapsulation principle of SOLID by exposing class fields directly. It may hinder future changes to the class. (Severity: medium)